### PR TITLE
fix: smallcaps compatible

### LINKED ISSUE #49: Ollinet Gov UI fails to resolve
**Describe the bug**
https://econ-gov.inter.trade/?agoricNet=ollinet fails to resolve and so is preventing product from changing auction params to carry out necessary liquidation bidding testing

devnet appears to resolve fine

**To Reproduce**

1. Go to https://econ-gov.inter.trade/?agoricNet=ollinet

**Expected behavior**
Should resolve, should be able to see Econ Gov UI

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

### PR DESCRIPTION
**Pull request by samsiegart**
fixes #49 

## Description

Makes the dapp work with smallcaps. I'm pretty confident the PSM stuff is still backwards compatible.

Also fixed the vault managers' vstorage paths since those were changed. Did not make that part backwards compatible, so no guarantees for devnet. We could consider pointing devnet users (and EC if worst-case mainnet is broken) to a different deployment, since we have one for every commit hash.

## Testing

Accepted invitations, made and voted on some proposals successfully. Did not test exhaustively with every type of proposal, but did some from both PSM and Vaults.

### CODE REVIEW

File: src/lib/makeImportContext.ts [hunk 1]
```diff
@@ -0,0 +1,384 @@
+// @ts-check
+// Copied and tweaked from @agoric/smart-wallet
```
Comment (turadg) [inline src/lib/makeImportContext.ts:2]: I suspect we can use without tweaking. If not we could give the SDK module some options for this use case.

Not worth slowing this down right now but consider putting an XXX tech debt comment to that effect.

File: src/lib/rpc.ts [hunk 2]
```diff
@@ -15,7 +21,10 @@ export const networkConfigUrl = (agoricNetName: string) => {
     return `https://${agoricNetName}.agoric.net/network-config`;
   }
 };
-export const rpcUrl = agoricNetSubdomain =>
```
Comment (turadg) [inline src/lib/rpc.ts:18]: Odd that the diff thinks this was removed. Consider adjusting line ordering so the line history is correct

File: src/lib/rpc.ts [hunk 3]
```diff
@@ -241,24 +90,29 @@ export const makeRpcUtils = async ({ agoricNet }) => {
...
+        path,
+        blockHeight,
+      ),
+  };
 
   return {
     agoricNames,
     follow,
     leader,
-    fromBoard,
```
Comment (turadg) [inline src/lib/rpc.ts:261]: Good encapsulation

File: src/lib/rpc.ts [hunk 4]
```diff
@@ -15,7 +21,10 @@ export const networkConfigUrl = (agoricNetName: string) => {
     return `https://${agoricNetName}.agoric.net/network-config`;
   }
 };
-export const rpcUrl = agoricNetSubdomain =>
+
+export const marshal = makeImportContext().fromBoard;
```
Comment (turadg) [inline src/lib/rpc.ts:25]: This verb implies a function but this is a Marshaller that has functions. Please rename.

**Review by turadg — APPROVED** [review]
🙏 

I didn't step through all the encoding mechanism so am trusting in the manual testing.

File: src/components/VotePanel.tsx [hunk 5]
```diff
@@ -78,7 +78,7 @@ function VoteOnQuestions(props: {
         <VoteOnQuestion
           {...props}
           details={details}
-          key={details.counterInstance.boardId}
+          key={details.closingRule.deadline}
```
[the remark this answers is not in the document]
Comment (samsiegart) [reply src/components/VotePanel.tsx:81]: We don't have the boardId anymore unless we serialize it back and look at the slots, so I figured this was a decent alternative. I don't think it's likely enough to worry about

File: src/lib/makeImportContext.ts (hunk 1 above)
Comment (samsiegart) [reply src/lib/makeImportContext.ts:2]: Done. I had trouble getting this to import, I think the other deps were giving it older versions of @agoric/vats and @agoric/store that broke, and I didn't want to fry the fish of upgrading all the dependencies yet to fix it.

Also, we would have had to patch to allow null slots like dapp-inter does.

File: src/lib/rpc.ts (hunk 2 above)
Comment (samsiegart) [reply src/lib/rpc.ts:18]: Attempted, will see if that worked

File: src/lib/rpc.ts (hunk 4 above)
Comment (samsiegart) [reply src/lib/rpc.ts]: I think it's consistent with @endo/marshal:

https://github.com/endojs/endo/tree/master/packages/marshal#usage

You call "makeMarshal" and it gives you an object with `serialize` and `unserialize`. Why isn't it `makeMarshaller`?

**Pull request merged**